src/main/groovy/edu/oregonstate/mist/hr/db/HRMockDAO.groovy

```diff
@@ -123,7 +123,7 @@ class HRMockDAO extends BaseHRDAO implements HRDAO {
         !invalidBusinessCenters.contains(businessCenter)
     }
 
-    List<Location> getLocations(String state) {
+    List<Location> getLocations(String state, String name) {
```

Comment:
Maybe I just don't understand groovy but I see two parameters (state, name) that aren't being used. Why is this?

Reply:
@Ruefa no worries. It's just a Java OOP stuff.

Indeed, `HRMockDAO` didn't use these two parameters for this function, however, since this class was designed to [extend `BaseHRDAO` and need to implement `HRDAO` interface](https://github.com/osu-mist/hr-api/blob/8986a75fc7106a11ea4eaff17f36ff1a0e64ca6/src/main/groovy/edu/oregonstate/mist/hr/db/HRMockDAO.groovy#L7) (good catch though, I forgot to update `BaseHRDAO` although it was overrided anyway. Fix has pushed.), based on the interface definition [here](https://github.com/osu-mist/hr-api/blob/1652b886dbe43f22ac42aba77f27940d9f312abd/src/main/groovy/edu/oregonstate/mist/hr/db/HRDAO.groovy#L31-L32), these two parameters will be used as binding variables for the SQL query.

Reply:
Ah I didn't see it was implementing an interface. Looks good then!